src/og/renderer/RendererEvents.js

```diff
@@ -264,29 +264,30 @@ class RendererEvents extends Events {
      * Set render event callback.
      * @public
      * @param {string} name - Event name
+     * @param {string} id - Event identifier
      * @param {eventCallback} callback - Callback function
-     * @param {number} [key] - Key code from og.input
+     * @param {number} [keyCode] - Key code from og.input
      * @param {*} sender - Callback context
      * @param {number} [priority] - Event callback priority
      */
-    on(name, p0, p1, p2, p3) {
+    on(name, callback, keyCode, sender, priority, id) {
         if (name === "keypress" || name === "charkeypress" || name === "keyfree") {
-            this._keyboardHandler.addEvent(name, p2, p1, p0, p3);
+            this._keyboardHandler.addEvent(name, callback, keyCode, sender, priority, id);
         } else {
-            super.on(name, p0, p1, p2);
+            super.on(name, callback, keyCode, sender);
```

Comment:
Third parameter is a sender in the parent class

Reply:
...this is why parameters called p1,p2 etc

Reply:
@program-sam take a look above, I think that the separate keyboard event method convenient way to refactor this "facade"